apps/chat-e2e/src/assertions/baseAssertion.ts

```diff
@@ -145,4 +145,48 @@ export class BaseAssertion {
       .soft(style[0], ExpectedMessages.elementColorIsValid)
       .toBe(expectedColor);
   }
+
+  public async assertStringTruncatedTo160(
```

Comment:
do we need this method, it is not used?

Reply:
It is used by the commented step - the testcase is blocked by an issue

Reply:
Moreover - that is a common check, that we can reuse in other situations, assuming we always have that restriction of 160 symbols max